packages/manager/modules/pci/src/projects/project/baremetal/instance/soft-reboot/soft-reboot.routing.js

```diff
@@ -0,0 +1,21 @@
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider
+    .state('pci.projects.project.baremetal.instance.soft-reboot', {
+      url: '/soft-reboot',
+      views: {
+        modal: {
+          component: 'pciInstancesInstanceReboot',
+        },
+      },
+      layout: 'modal',
+      translations: {
+        value: ['../reboot'],
```

Comment:
Either the component should load its own translations or the module should have its own but I'm not sure we should add translations dependencies between modules

Reply:
The component loads it's own translations and this is not required and has been removed. Thanks!